Client/Frontend/UIConstants.swift

```diff
@@ -9,7 +9,7 @@ public struct UIConstants {
     static let AboutHomePage = URL(string: "\(WebServer.sharedInstance.base)/about/home/")!
 
     static let AppBackgroundColor = UIColor.black
-    static let SystemBlueColor = UIColor(red: 0 / 255, green: 122 / 255, blue: 255 / 255, alpha: 1)
+    static let SystemBlueColor = UIColor(rgb: 0x0297F8)
```

Comment:
We use this blue throughout the app (home panels, login manager, action overlay). Was the intention to change it in all of these places?